arrow-libs/suspendapp/suspendapp/src/jsMain/kotlin/arrow/continuations/Enviroment.js.kt

```diff
@@ -55,13 +56,16 @@ public object JsProcess : Process {
               delay(1.hours)
             }
           }
-        runCatching { block(innerScope) }.also { keepAlive.cancelAndJoin() }.getOrThrow()
+        runCatching { withContext(context, block) }
+          .also { keepAlive.cancelAndJoin() }
+          .getOrThrow()
       }
       .startCoroutine(Continuation(EmptyCoroutineContext) {})
```

Comment:
should this be `.startCoroutine(Continuation(innerScope.coroutineContext) {})`?

Reply:
`innerScope.coroutineContext == EmptyCoroutineContext`, I guess I thought using it directly was more readily.